### ISSUE TITLE
Better generated documentation style

### ISSUE THREAD

**Issue by davidcoallier** [frapi/frapi#5]
I'd like if someone could look into making the generated PDF documentation nicer (Any generated docs even)

**Comment by Theaxiom** [comment]
Shouldn't you be able to do that by modifying the view templates? I have not looked much into frapi yet, but I believe it is MVC, so you just have to modify the HTML.

**Comment by davidcoallier** [comment]
Yes :) But it has to be done. Right now this code is tightly bounded in the admin part and it simply has to be made better (Better styles, better PDF look generation)
